app/android/src/uk/co/lutraconsulting/InputActivity.java

```diff
@@ -16,6 +16,10 @@
 import android.util.Log;
 import android.os.Bundle;
 import android.os.Build;
+import android.os.VibratorManager;
```

Comment:
This `VibratorManager` seems to be available from SDK 31 https://developer.android.com/reference/android/os/VibratorManager